meetup/services/generic_api.py

```diff
@@ -0,0 +1,21 @@
+from django.http import HttpResponse
+import requests
+from meetuppizza.settings import base
```

Comment:
Importing your base settings directly is a smell. The idea of separating your settings files into base/dev/production/etc. is to allow Django to construct the correct settings at runtime. This is a little like importing an abstract class and calling methods on it without creating a concrete implementation. Instead, you should always import and use `django.conf.settings` to access your settings values. Depend on abstractions, not concrete details.